api/cosmos/base/query/v1beta1/pagination.pulsar.go

```diff
@@ -1124,10 +1124,10 @@ const (
 // PageRequest is to be embedded in gRPC request messages for efficient
 // pagination. Ex:
 //
-//  message SomeRequest {
```

Comment:
we should ignore all autogenerated files

Reply:
ok -- for expedience sake I'm going to see which pulsar files were touched and check them out of main.

Reply:
thanks